### PR TITLE
{bio}[system/system] CellRanger v7.0.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by branfosj — DISMISSED** [review]

File: easybuild/easyconfigs/c/CellRanger/CellRanger-7.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+version = '7.0.0'
+
+homepage = 'https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger'
+description = """Cell Ranger is a set of analysis pipelines that process Chromium
+ single-cell RNA-seq output to align reads, generate gene-cell matrices and perform
+ clustering and gene expression analysis."""
+
+toolchain = SYSTEM
+
+# Download manually from https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/latest
```
[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/c/CellRanger/CellRanger-7.0.0.eb:16]: ping @boegel

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/8f223202cbd2d1494021a4a9e3a2a820 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Pull request merged**